src/components/categories/weather/models/TimeDropdown.vue

```diff
@@ -17,17 +18,29 @@
 </template>
 <script>
 
+    import moment from 'moment';
+
     export default {
         name: 'TimeDropdown',
         components: {},
         data() {
             return {
-                context: null
+                context: null,
+                labels: {
+                    labelNoDateSelected: 'Nie wybrano daty',
+                }
             }
         },
         methods: {
-            setDay(){
-                this.$root.$emit('showWeatherMessage', [this.context.selectedFormatted,this.context.activeYMD]);
+            setDay() {
+
+                if (this.context.selectedFormatted === "No date selected") {
```

Comment:
Zamiast no date selected wartość z $t:)

Reply:
Ale to jest wartość początkowa z kalendarza, ustawienie jest z $t w innym przypadku zepsuje działanie logiki